tests/upgrades/test_errata.py

```diff
@@ -253,15 +235,14 @@ def test_post_scenario_errata_count_installation(self, pre_upgrade_data):
         # waiting for errata count to become 0, as profile uploading take some
         # amount of time
         wait_for(
-            lambda: self._errata_count(ak=activation_key) == 0,
+            lambda: host.read().content_facet_attributes['errata_counts']['total'] == 0,
             timeout=400,
             delay=2,
             logger=logger,
         )
-        host = entities.Host().search(query={'search': f'activation_key={activation_key}'})[0]
-        assert host.content_facet_attributes['errata_counts']['total'] == 0
-        for package in FAKE_9_YUM_UPDATED_PACKAGES:
-            install_or_update_package(client_hostname=client_container_id, package=package)
+        rhel_client.execute(
+            f'yum -y install {" ".join(FAKE_9_YUM_UPDATED_PACKAGES)}'
+        )  # why do we install packages at the end of the test?
 
 
 class TestScenarioErrataCountWithPreviousVersionKatelloAgent(TestScenarioErrataAbstract):
```

Comment:
Is this test class still valid considering that we have moved from Satellite Tools to Satellite Client? There is not much sense in re-installing katello-agent package on the content host after we perform the Satellite upgrade.

Reply:
my vote is to kill it!!!

Reply:
I have removed this test class based on our discussion in the JPL Team meeting and based on the discussion with the Phoenix team in the #satellite-phoenix channel.